.travis.yml

```diff
@@ -0,0 +1,22 @@
+language: go
+
+go:
+  - 1.6
+  - tip
+
+before_install:
+  - go get github.com/Masterminds/glide
+  - go get golang.org/x/tools/cmd/cover
+  - go get github.com/golang/lint/golint
```

Comment:
Yes, probably. But at this point I wouldn't bother with that (we don't have any tests yet anyway).